external/fv3fit/fv3fit/_shared/pytorch/graphPredict.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
```

Comment:
Please rename all files to snake_case, e.g. graph_predict.py, and update associated imports with the new names.